airbyte-integrations/connectors/source-google-ads/source_google_ads/source.py

```diff
@@ -56,24 +74,37 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
 
     def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         google_api = GoogleAds(credentials=self.get_credentials(config), customer_id=config["customer_id"])
+        account_info = self.get_account_info(google_api)
+        time_zone = self.get_time_zone(account_info)
         incremental_stream_config = dict(
-            api=google_api, conversion_window_days=config["conversion_window_days"], start_date=config["start_date"]
+            api=google_api,
+            conversion_window_days=config["conversion_window_days"],
+            start_date=config["start_date"],
+            time_zone=time_zone
         )
 
-        custom_query_streams = [
-            CustomQuery(custom_query_config=single_query_config, **incremental_stream_config)
-            for single_query_config in config.get("custom_queries", [])
-        ]
-        return [
-            UserLocationReport(**incremental_stream_config),
-            AccountPerformanceReport(**incremental_stream_config),
-            DisplayTopicsPerformanceReport(**incremental_stream_config),
-            DisplayKeywordPerformanceReport(**incremental_stream_config),
-            ShoppingPerformanceReport(**incremental_stream_config),
-            AdGroupAdReport(**incremental_stream_config),
+        streams = [
             AdGroupAds(api=google_api),
             AdGroups(api=google_api),
             Accounts(api=google_api),
             Campaigns(api=google_api),
             ClickView(**incremental_stream_config),
-        ] + custom_query_streams
+        ]
+
+        custom_query_streams = [
```

Comment:
a user could configure a custom query involving metrics with a manager account. Is it possible to check this here too, and exclude custom query streams from manager accounts if they contain metrics?

Reply:
add `is_metrics_in_custom_query` in check method